api/storage/v1alpha1/vulnerabilityreport_types.go

```diff
@@ -156,12 +156,15 @@ type Vulnerability struct {
 	// CVSS scoring details
 	CVSS map[string]CVSS `json:"cvss,omitempty" protobuf:"bytes,12,rep,name=cvss"`
 
+	// CWEs with which the CVE is classified
+	CWES []string `json:"cwes,omitempty"  protobuf:"bytes,13,rep,name=cwes"`
```

Comment:
```suggestion
	CWEs []string `json:"cwes,omitempty"  protobuf:"bytes,13,rep,name=cwes"`
```
The right field name here is `CWEs`